dbsc/workflow/rules/mrtpipelines.smk

```diff
@@ -130,8 +130,8 @@ rule dwi2response:
         dwi=rules.nii2mif.output.dwi,
         mask=rules.nii2mif.output.mask,
     params:
-        shells=f"-shells {shells}" if shells else "",
-        lmax=f"-lmax {lmax}" if lmax else "",
+        shells=f"-shells {','.join(shells)}" if shells else "",
```

Comment:
Loking at the help text (and `nargs`) of the `--shells` argument, it needs to be there, right? Maybe worth erroring out/raising an exception somewhere if this isn't defined.

Reply:
Doesn't have to be required (similar to `lmax`).